core/src/main/java/de/danoeh/antennapod/core/service/playback/FeedVolumeReduction.java

```diff
@@ -0,0 +1,16 @@
+package de.danoeh.antennapod.core.service.playback;
+
+import de.danoeh.antennapod.core.feed.FeedPreferences;
+import de.danoeh.antennapod.core.feed.VolumeReductionSetting;
+
+public class FeedVolumeReduction {
+    float getReductionFactor(FeedPreferences preferences) {
+        VolumeReductionSetting volumeReductionSetting = preferences.getVolumeReductionSetting();
```

Comment:
What about using `getVolumeReductionSetting().getReductionFactor()` instead of a new class?

Reply:
You suggest moving the method to `VolumeReductionSetting`? I like the idea.

Reply:
Yes. You could even add a second constructor parameter to the enum. Then the switch is no longer needed.